build_tools/cmake/iree_llvm.cmake

```diff
@@ -99,13 +99,15 @@ macro(iree_llvm_configure_installed)
   set(IREE_CLANG_BINARY "${LLVM_INSTALL_DIR}/llvm/bin/clang${CMAKE_EXECUTABLE_SUFFIX}")
   string(REGEX REPLACE "[^0-9].*" "" _LLVM_VERSION_MAJOR "${LLVM_VERSION}")
   set(IREE_CLANG_BUILTIN_HEADERS_PATH "${LLVM_INSTALL_DIR}/llvm/lib/clang/${_LLVM_VERSION_MAJOR}/include/")
+
+  set(IREE_LLVM_CPU_TARGETS "${LLVM_TARGETS_TO_BUILD}")
```

Comment:
Why add a second variable that always matches the first? Can you just use `LLVM_TARGETS_TO_BUILD` everywhere that `IREE_LLVM_CPU_TARGETS` is used?

(and keep the name for `IREE_DEFAULT_CPU_LLVM_TARGETS`)